packages/framework-provider-kubernetes-infrastructure/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "esModuleInterop": true,
```

Comment:
Another package that loses ESModules support 😂 

Just joking, this one is deprecated so it doesnt matter